Update windows install steps (#3571)
SFDO-Tooling/CumulusCI: docs/get-started.md

Open your preferred terminal application (such as
[cmd.exe](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd)
on Windows). If your terminal is already open, close it and reopen it.
Enter the following commands:

```console
$ python -m pip install --user pipx
$ python -m pipx ensurepath
```

> Close your current cmd window and open a new one verify that `pipx` was successfully installed:

Reviewer: ```suggestion
Close your current command prompt and open a new one to verify that `pipx` was successfully installed:
```

Look for a version number after entering this command, such as:
`0.12.3.1`.

If you encounter an error, such as
`'pipx' is not recognized as an internal or external command, operable program or batch file.`,

> you can try [updating your environment variables manually](update-environment-variables-manually).

Reviewer: ```suggestion
you can try [updating your environment variables manually](#update-environment-variables-manually).
```

When finished, [verify your installation](verify-your-installation).

#### Update Environment Variables Manually

1.  Click Start and search for `edit environment variables`, or open
    System properties and click `Advanced system settings`.
2.  Click the `Environment Variables` button.
3.  To change System variables, you need non-restricted access
    (administrator rights) to your machine. Add these paths to your PATH
    environment variable:
    -   `%USERPROFILE%\.local\bin`
    -   `%USERPROFILE%\AppData\Roaming\Python\Python38\Scripts`

![image](images/env-var2.png)

```{note}
Be precise when entering these paths. Add them at the very end of the
Variable Value already in place. Separate each path by a semicolon (;)
with no space between path names.
```

> 

Reviewer: 😕 
```suggestion
(verify-your-installation)=
```

When finished, [verify your installation](verify-your-installation).

#### Update Environment Variables Manually

1.  Click Start and search for `edit environment variables`, or open
    System properties and click `Advanced system settings`.
2.  Click the `Environment Variables` button.
3.  To change System variables, you need non-restricted access
    (administrator rights) to your machine. Add these paths to your PATH
    environment variable:
    -   `%USERPROFILE%\.local\bin`
    -   `%USERPROFILE%\AppData\Roaming\Python\Python38\Scripts`

![image](images/env-var2.png)

```{note}
Be precise when entering these paths. Add them at the very end of the
Variable Value already in place. Separate each path by a semicolon (;)
with no space between path names.
```

> (verify-your-installation)=

Reviewer: Sorry, linter failure:
```suggestion

(verify-your-installation)=

```
